### PR TITLE
Header-portfolio

### PR DESCRIPTION
**Pull request by rbd3**
I finish the portfolio mobile version project

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+
+    <header class="app-header">
+        <a href="#" class="logo">Narson</a>
+        <nav class="ambony">
+            <a href="#" class="humburger">☰</a>
```
Comment (mukhammadrizooff) [inline index.html]: - [ ] [REQUIRED] Good job so far! Please add the correct style to your `Hamburger` button like [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0-1&t=04rBLu4GidVN0Uk6-0) 🙏🏼 

image

File: index.html [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+            <br>Glad to see you!
+        </h1>
+        <p class="detail">
+            I'm a software developer! I can 
+            help you build a product , feature or website Look through some of my work and experience! 
+            If you like what you see and have a project you need coded, don’t hesitate to contact me.
+        </p>
+        <a href="#" class="connection"><h4>LET'S CONNECT</h4></a>
+
+        <ul class="social">
```
Comment (mukhammadrizooff) [inline index.html:35]: - [ ] [REQUIRED] Good job so far! Please set correct color(`#505F79`) to your social network icons 🙏🏼 

image

**Review by mukhammadrizooff — CHANGES_REQUESTED** [review]
Hi @rbd3, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- No linter errors ✔️ 
- Correct GitHub Flow ✔️ 
- Good implementations ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+            <a href="https://github.com/rbd3"><li class="github">
+                <img src="media/github.svg" alt="github" title="github" />
+            </li></a>
+            <a href="#"><li class="angellist">
+                <img src="media/angellist.svg" alt="Angellist" title="Angellist" />
+            </li></a>
+            <a href="#"><li class="medium">
+                <img src="media/medium.svg" alt="Medium" title="Medium" />
+            </li></a>
+        </ul>
```
Comment (Shedrack-Sunday) [inline index.html]: - [ ] As asked by the previous reviewer, kindly adjust the social links to that of the Figma design. The given color is `background: #505F79;`. This has to be specific 👍 

## View:
image

## figma:
image

**Review by Shedrack-Sunday — CHANGES_REQUESTED** [review]
Hi @rbd3      👋 

# CHANGES REQUIRED ♻️

Great work on the project 👏
You've done well implementing the requirements,  However, there are a few you need to work on, to move to the next project. 💯

 


## Highlights:
✔️ good PR summary
✔️ Changes taken note of

  
# Required Changes ♻️

- [x] I must admit you have really put in the effort so far in this project, however, the changes required below need to be implemented :
- The color of the social links is still not adjusted 


_check the comment section for a detailed review_ 

# Optional suggestions 

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html (hunk 3 above)
Comment (rbd3) [reply index.html]: hi @Shedrack-Sunday ,
I don't understand what you asking me to do. I already change my social color here '.social {
  color: #505f79;
} '

Comment (rbd3) [reply index.html]: @Shedrack-Sunday 
Do you see my question above?

Comment (Shedrack-Sunday) [reply index.html]: Hello @rbd3, it's still not reflecting on your project. If you believe it does so on your end, please have a snapshot and send it here. 
- I specifically cloned your project, and I can't  see the difference. 
- Another reason might be that you have a shadowing effect around the social links container. Please kindly check and fix that ✅

Comment (rbd3) [reply index.html]: @Shedrack-Sunday 
I changed the color of my social icons image like this ' .twitter, .linkindin, .github, .medium, .angellist { color: #505f79;}. But I don't see a change on my page.

**Comment by rbd3** [comment]
Hi,
I changed some code on the social icon.
[image: social-icon-code.png]
[image: social-icon.png]

Le mer. 29 mars 2023 à 13:01, Shedrack Sunday ***@***.***> a
écrit :

**Review by IngDavidTM — APPROVED** [review]
Hi @rbd3 🙋‍♂️, 

# ✔️   Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations!  🎉💯🌟 

## Top highlight 🎯

- The Pull Request follows the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- Linters are working properly ✅ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**